examples/go-union-type/README.md

```diff
@@ -0,0 +1,28 @@
+# Go Union Types
+
+Modelina now supports union types for `Go`. Since `Go` does not have native union types support modelina uses `struct embeddings` to mock union types.
+
+```go
+type Union struct {
+  CustomStruct
+  int
+  string
+  ModelinaArrType []string
+  ModelinaDictType map[string] interface{}
+  ModelinaAnyType interface {}
+}
+```
```

Comment:
```suggestion
```

Lets not add the output to the readme, cause they get outdated quickly and forgotten about

Reply:
I was trying to add a quick example of how all the data types would look like.

Reply:
Yes, but, code change, and docs rarely follow, which is why you normally don't see any code in the docs, but links to examples that has it.